Added mobilenet_v1 folder

hitesh-anand opened the pull request:
the file named "mobilenet-v1" was added accidentally.
The main files are inside the folder named "mobilenet_v1"

hitesh-anand committed:
Create mobilenet-v1 folder

hitesh-anand committed:
Create README.txt

hitesh-anand committed:
Added two .ipynb files 

two notebook files containing two mobilenet v1 models

SomTambe commented on classification/mobilenet_v1/README.txt:
```diff
@@ -0,0 +1,37 @@
+MOBILE NET v1 (TENSORFLOW)
+					(MODEL ZOO)
+
+
```
Rewrite the README in Markdown. Let it be consistent with the rest of the repo.

SomTambe commented on classification/mobilenet-v1:
```diff
@@ -0,0 +1 @@
+
```
Why can't you delete this file?

SomTambe reviewed (changes requested):
It was informed well before that jupyter notebooks won't be accepted. Please help us in maintaining consistency with the rest of the repo. Others have done it too.

hitesh-anand committed:
Delete classification/mobilenet_v1 directory

hitesh-anand committed:
Create README.txt

hitesh-anand committed:
Added model.py and README file

hitesh-anand committed:
Delete README.txt

hitesh-anand committed:
Delete mobilenet-v1

hitesh-anand committed:
Add files via upload

hitesh-anand committed:
Delete model.ipynb

hitesh-anand commented:
Deleted the earlier added Jupyter notebook files and README, added a new README written using markdown and a .py file containing the mobilenet model implemented in Google Collab.

namangup commented:
Look at the pytorch version of MobileNet to get an idea of what is expected. Lot of changes required.
